cli/compose/loader/volume.go

```diff
@@ -70,6 +70,8 @@ func populateFieldFromBuffer(char rune, buffer []rune, volume *types.ServiceVolu
 		switch option {
 		case "ro":
 			volume.ReadOnly = true
+		case "rw":
```

Comment:
@dnephin:
> I think we should only support those in the expanded format.

That'll cause confusion, it should support the exact same set of options supported by `docker run`:

```
  -v, --volume value                  Bind mount a volume (default []). The format
                                      is `[host-src:]container-dest[:<options>]`.
                                      The comma-delimited `options` are [rw|ro],
                                      [z|Z], [[r]shared|[r]slave|[r]private],
                                      [delegated|cached|consistent], and
                                      [nocopy]. The 'host-src' is an absolute path
                                      or a name value.
```

https://github.com/docker/docker/blob/3de58eb2bca720be0f5940cdd928de7a96220174/docs/reference/commandline/run.md

@thaJeztah Yeah, it's a bigger issue than just this one `rw` flag. It's about matching behavior, because the options used to be passed on and they worked. So it's actually a BC break.